cron.yaml

```diff
@@ -3,26 +3,6 @@
 # To manually deploy this config, run:
 #   gcloud app deploy --project flutter-dashboard cron.yaml
 cron:
-- description: refresh chromebot build status (cocoon)
-  url: /api/refresh-chromebot-status?repo=cocoon
```

Comment:
Can we remove these handlers as well?

Reply:
Yeah, I will remove after couple of days validation.